packages/react-native/src/notifier.js

```diff
@@ -80,6 +80,10 @@ const _createClient = (opts, jsOpts) => {
     bugsnag.setContext(opts.context)
   }
 
+  if (opts.featureFlags && opts.featureFlags !== opts._originalValues.featureFlags) {
+    NativeClient.addFeatureFlags(opts.featureFlags)
```

Comment:
Should this be `bugsnag.addFeatureFlags` with `@bugsnag/plugin-react-native-client-sync` handling the call to the native client?

Reply:
Yes it probably should be 👍